pyeudiw/satosa/backend.py

```diff
@@ -361,7 +376,9 @@ def _request_endpoint_dpop(self, context, *args) -> Union[JsonResponse, None]:
 
             # take WIA
             wia = unpad_jwt_payload(context.http_headers['HTTP_AUTHORIZATION'])
```

Comment:
please take context.http_headers['HTTP_AUTHORIZATION'] in a variable and remove the "DPOP " prefix accordingly

Reply:
Resolved in [895a19b](https://github.com/italia/eudi-wallet-it-python/pull/94/commits/895a19b6f589ca6d38df8e90ff3331fc56a9bbab)